extensions/pkg/util/shoot_test.go

```diff
@@ -71,4 +73,43 @@ var _ = Describe("Shoot", func() {
 			Expect(err).NotTo(HaveOccurred())
 		})
 	})
+
+	Describe("IsPSPDisabled", func() {
+		var shoot = &gardencorev1beta1.Shoot{
+			Spec: gardencorev1beta1.ShootSpec{
+				Kubernetes: gardencorev1beta1.Kubernetes{
+					KubeAPIServer: &gardencorev1beta1.KubeAPIServerConfig{},
+				},
+			},
+		}
+
+		It("should return true if PodSecurityPolicy admissionPlugin is disabled", func() {
```

Comment:
Can you please add a test if `kubeapiserver` is nil?

Reply:
Why?

Reply:
You had put a check if kubeapiserver is not nil in the helper function.

Reply:
it returns false, right? I feel that's not what this function is focused on anyway.